doc/README.md

```diff
@@ -0,0 +1,11 @@
+Documentation
```

Comment:
This file should have a copyright header (yeah, I know...but documentation counts too!)